floss/language/rust/extract.py

```diff
@@ -0,0 +1,164 @@
+# Copyright (C) 2023 Mandiant, Inc. All Rights Reserved.
+
+import sys
+import logging
+import pathlib
+import argparse
+from typing import List, Tuple, Iterable, Optional
+from collections import namedtuple
+
+import pefile
+import binary2strings as b2s
+from typing_extensions import TypeAlias
+
+from floss.results import StaticString, StringEncoding
+from floss.language.utils import find_lea_xrefs, get_struct_string_candidates
+
+logger = logging.getLogger(__name__)
+
+MIN_STR_LEN = 4
+
+VA: TypeAlias = int
+
+Strings = namedtuple("Strings", ["string", "start_address", "end_address"])
+
+
+def get_rdata_section_info(pe: pefile.PE) -> pefile.SectionStructure:
+    """
+    Retrieve info about .rdata section
+    """
+    rdata_structure = pefile.SectionStructure(pe.__IMAGE_SECTION_HEADER_format__)
+
+    for section in pe.sections:
+        if section.Name.startswith(b".rdata\x00"):
+            rdata_structure = section
+            break
+    else:
+        logger.error("No .rdata section found")
+        return rdata_structure
```

Comment:
should this be returned here and can you double check the initialization in line 30 is what you want in this function?